### ISSUE TITLE
不受支持的字符串类型传入numpy

### ISSUE THREAD

**Issue by Songshen1996** [shijiawei124/RPT#1]
>>Root Phenotype Toolbox v0.0.1
2025-10-03 22:15:45 >>Start training...
2025-10-03 22:15:46 >>[TRAIN]	
------------Environment Information-------------
platform: Windows-10-10.0.26100-SP0
Python: 3.9.10 (tags/v3.9.10:f2f3f53, Jan 17 2022, 15:14:21) [MSC v.1929 64 bit (AMD64)]
Paddle compiled with cuda: True
NVCC: Build cuda_11.2.r11.2/compiler.29373293_0
cudnn: 8.9
GPUs used: 1
CUDA_VISIBLE_DEVICES: None
GPU: ['GPU 0: NVIDIA GeForce']
GCC: gcc (MinGW.org GCC-6.3.0-1) 6.3.0
PaddlePaddle: 2.4.1
OpenCV: 4.10.0
------------------------------------------------
2025-10-03 22:15:46 >>[TRAIN]	train_list.txt is not empty
2025-10-03 22:15:47 >>[TRAIN]	
---------------Config Information---------------
batch_size: 2
iters: 200
loss:
  coef:
  - 1
  types:
  - coef:
    - 1
    - 1
    losses:
    - type: CrossEntropyLoss
    - type: DiceLoss
    type: MixedLoss
lr_scheduler:
  T_max: 200
  learning_rate: 0.0002
  type: CosineAnnealingDecay
model:
  pretrained: https://bj.bcebos.com/paddleseg/dygraph/mix_vision_transformer_b0.tar.gz
  type: ESegFormer_B0
optimizer:
  beta1: 0.9
  beta2: 0.999
  type: AdamW
  weight_decay: 0.01
test_config:
  aug_eval: true
  crop_size:
  - 1024
  - 1024
  flip_horizontal: true
  flip_vertical: true
  is_slide: true
  stride:
  - 768
  - 768
train_dataset:
  dataset_root: C:/Users/Songshen/Downloads/train_data_root-20250814T112714Z-1-002/train_data_root/train
  mode: train
  transforms:
  - max_scale_factor: 2.0
    min_scale_factor: 0.5
    scale_step_size: 0.25
    type: ResizeStepScaling
  - crop_size:
    - 512
    - 512
    label_padding_value: 0
    type: RandomPaddingCrop
  - type: RandomHorizontalFlip
  - prob: 0.1
    type: RandomBlur
  - brightness_range: 0.4
    contrast_range: 0.4
    saturation_range: 0.4
    type: RandomDistort
  - type: RandomNoise
  - type: Normalize
  type: RootSeg
val_dataset:
  dataset_root: C:/Users/Songshen/Downloads/train_data_root-20250814T112714Z-1-002/train_data_root/train
  mode: val
  transforms:
  - type: Normalize
  type: RootSeg
------------------------------------------------
2025-10-03 22:15:54 >>unsupported format string passed to numpy.ndarray.__format__
2025-10-03 22:15:54 >>Train Done
